sentry/values.yaml

```diff
@@ -136,11 +136,15 @@ postgres:
   port: 5432
 
 rabbitmq:
+  # either use RabbitMQ or Redis as broker
+  enabled: true
   username: guest
   password: guest
   host: rabbitmq-ha
 
 redis:
+  # either use RabbitMQ or Redis as broker
+  enabled: false
```

Comment:
This implies that Redis might not be needed.
Redis is always needed.

You can remove that and put `if not .Values.rabbitmq.enabled` as a condition.

Reply:
Removed